src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisResultContext.java

```diff
@@ -70,14 +68,37 @@ public static SecurityAnalysisResultContext fromMessage(Message<String> message,
                 provider,
                 parameters,
                 new ReportInfos(reportUuid, reporterId, reportType),
-                userId
+                userId,
+                limitReductions
         );
         return new SecurityAnalysisResultContext(resultUuid, runContext);
     }
 
+    private static List<List<Double>> getLimitReductionsFromHeader(MessageHeaders headers, String name) {
+        String header = (String) headers.get(name);
```

Comment:
Why a string ?
Use `List<List<Double>>` ?

Reply:
done in https://github.com/gridsuite/security-analysis-server/pull/134/commits/4d7a9ae946e46b1f9d39f35857bbe74c1fdc3b00